dashboard/config/scripts/allthethings.script

```diff
@@ -116,8 +116,8 @@ stage 'Studio'
 level 'Resizing Test'
 
 stage 'Long assessment'
-assessment 'Test longassessment levelgroup'
-assessment 'Test longassessment levelgroup split'
+assessment 'Test longassessment levelgroup new'
+assessment 'Test longassessment levelgroup split new'
```

Comment:
Unfortunately anyone who had progress in one of the LevelGroup levels would get a 5xx exception when loading their last attempt.  It'd try to load the old `LevelSource` data (which is an array, not an object) and throw an exception.

Changing the names forces the levels to get assigned new IDs, which invalidates the old `LevelSource` data.  The data will still be there, but it won't match up with any current level IDs.